runtime/queries/yuck/highlights.scm

```diff
@@ -1,66 +1,107 @@
+; Errors
+
 (ERROR) @error
 
-(line_comment) @comment
+; Comments
+
+(comment) @comment
+
+; Operators
+
+[
+  "+"
+  "-"
+  "*"
+  "/"
+  "%"
+  "||"
+  "&&"
+  "=="
+  "!="
+  "=~"
+  ">"
+  "<"
+  ">="
+  "<="
+  "!"
+  "?."
+  "?:"
+] @operator
+
+(ternary_expression
+  ["?" ":"] @operator)
+
+; Punctuation
 
-; keywords and symbols
+[ ":" "." "," ] @punctuation.delimiter
 
-(keyword) @keyword
-(symbol) @tag
+[ "{" "}" "[" "]" "(" ")" ] @punctuation.bracket
 
-; literals
+; Literals
 
-(bool_literal) @constant.builtin.boolean
-(num_literal) @constant.numeric
+(number (float)) @constant.numeric.float
+
+(number (integer)) @constant.numeric.integer
+
+(boolean) @boolean
```

Comment:
```suggestion
(boolean) @constant.builtin.boolean
```

Reply:
Oh well, thanks for the catches, I guess I missed these while converting from neovim to helix again (bigger PR in upstream stirred up the highlights a little bit)...